apps/passport-client/src/localstorage.ts

```diff
@@ -72,3 +72,11 @@ export function loadIdentity(): Identity | null {
 export function saveIdentity(identity: Identity): void {
   window.localStorage["identity"] = identity.toString();
 }
+
+export function saveParticipantInvalid(participantInvalid: boolean) {
+  window.localStorage["participantInvalid"] = participantInvalid;
```

Comment:
why do we need to save this to localStorage?

Reply:
I save it to localstorage so that when the passport decides it's invalid, it can show this popup immediately if the user happens to refresh, without having to load the user from the internet, so that the page doesn't flicker.